.devcontainer/devcontainer.json

```
@@ -0,0 +1,11 @@
+{
+  "name": "Bowtie",
+  
+  "dockerFile": ".codespace.Dockerfile",
+
+  "runArgs": ["-v", "/var/run/docker.sock:/var/run/docker.sock"],
+
+  "onCreateCommand": "sudo chown -R vscode /var/run/docker.sock; pip install -r requirements.txt -e .",
+  
+  "postCreateCommand": "for each in $(ls implementations); do docker pull ghcr.io/bowtie-json-schema/$each; sudo cp /workspaces/bowtie/docs/gitpod-motd.txt /usr/local/etc/vscode-dev-containers/first-run-notice.txt; done;"
```

```suggestion
  "postCreateCommand": "for each in $(ls implementations); do docker pull "ghcr.io/bowtie-json-schema/$each:latest"; sudo cp /workspaces/bowtie/docs/gitpod-motd.txt /usr/local/etc/vscode-dev-containers/first-run-notice.txt; done;"
```

Also probably you can rename `gitpod-motd` to just be `motd` or something more generic, so that it's clear it's used for both.